tests/playwright/sanity/context-menu.test.ts

```diff
@@ -62,7 +62,7 @@ test.describe( 'Context menu', () => {
 		await wpAdmin.openNewPage();
 		await editor.addWidget( 'heading' );
 		await editor.openElementsPanel();
-		await contextMenu.selectWidgetContextMenuItem( 'heading', 'Navigator' );
+		await contextMenu.selectWidgetContextMenuItem( 'heading', 'Structure' );
```

Comment:
Shouldn't this be a conditional? 'Structure' when top bar active, 'Navigator' otherwise?

Reply:
No - the test always run with the top bar experiment switched on.